clean-language/README-copy.md

```diff
@@ -0,0 +1,146 @@
+(A) It absorbs all the knowledge like water.
+(Q)And when it absorbs all the knowledge what happens next?
+(A) And then it is full and water starts to spill over and it cannot absorb anymore; and then, over time, it dries out...
+(Q) What would you like to have happen?
+(A) I would like my knowledge stay with me instead of drying out.
+And I want to keep learning indefinitely without being limited by saturated sponge.
+
+This is the moment of deep realization of something profound about my way of learning.
+This is exactly what I experience, I am a lifelong learner, but the knowledge tends to dry out over time, sometimes to the point that I can read the same book as if for the first time.
+This kind of epiphany makes me want to change my metaphor.
+Before I was proud of being a sponge-learner, now I see it's limitations.
+Action: add about opinion of the metaphor of the others.
+By exploring metaphors of other people I help them understand themselves better and also get a view into their worlds and can upgrade my own metaphors.
+
+## Applications of clean language
+There are numerous applications of clean language.
+It is a perfect tool for one-on-one coaching and is often used for executive coaching.
+Using clean questions, the coach helps their clients discover the limiting patterns and beliefs that hold them back, reinforce positive beliefs, make unconscious knowledge conscious.
+Client also get to experience an exquisite attention to their words.
+It is unique in itself as usually people listen to respond and the act of listening becomes a rare phenomenon.
+
+Clean language can also be applied in a team setting and be used for conflict resolution, team development and motivation, joint understanding and focus, building deeper connection.
+Every team is going thru number of development stages: forming, storming, norming and performing.
+The most challenging one and emotionally draining is storming.
+During this phase team members tend to disagree a lot and conflicts are not uncommon.
+It is possible to minimize the duration of this stage by equipping the teams with the clean questions as a conflict resolution technique that will bring them closer into their optimal performance.
+
+My favorite part of clean language and technique I used most often is a clean setup.
+It is very simple, it only has 3 questions:
+(I am not sure if I want to put the questions in here)
+For ...
+to go just as you'd like, it will be like what?
+You'd like to be like what?
+What support or resources do you need?
+I use it for various meetings to set up the stage and account for the desires and expectations of each participant.
+What is the most amazing about it is that most of the time it goes just as we'd like and we get exactly what we wanted.
+It keeps consistently delivering a desired result.
+
+## Contracts?
+Do we want to talk about contracts here? Don't use it without an explicit agreement.
```

Comment:
Also, maybe suggest to have an easrly experience with clean language with someone who is experience using it.

Reply:
@joelgrimberg commits to fix this :)

Reply:
done